homeassistant/components/netgear/__init__.py

```diff
@@ -1 +1,90 @@
-"""The netgear component."""
+"""Support for Netgear routers."""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_SSL,
+    CONF_USERNAME,
+)
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import DOMAIN, PLATFORMS
+from .router import NetgearRouter
+
+_LOGGER = logging.getLogger(__name__)
+
+NETGEAR_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_HOST): cv.string,
+        vol.Optional(CONF_PORT): cv.port,
+        vol.Optional(CONF_SSL, default=False): cv.boolean,
+        vol.Optional(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+    }
+)
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
We shouldn't move the config YAML, since it's legacy. We should just try to import the old config to the config flow. So remove the config schema here.